tas/views.py

```diff
@@ -79,17 +92,22 @@ def profile_edit(request):
     if request.method == 'POST':
         request_pi_eligibility = request.POST.get('request_pi_eligibility')
         form = UserProfileForm(request.POST, initial=tas_user)
+        '''
+            if user is federated only check/update PI Eligibility Requests
+        '''
+        if is_federated and request_pi_eligibility:
```

Comment:
I think we probably still want to put this under `if form.is_valid()` - that likely performs CSRF checking, yes?

Reply:
I think the CSRF check happens in the middleware, https://docs.djangoproject.com/en/1.11/ref/csrf/#how-to-use-it